### DISCUSSION TITLE
How do I change the proxy config

### DISCUSSION THREAD

**Discussion by NubeBuster** [coollabsio/coolify#3602] [Q&A] [answered]
I am running a docker registry and some layers are too big for the proxy. I need to change the max allowed size property of the proxy config. Where do I do that?

It needs to be something like
traefik.http.middlewares.my-requests.maxRequestBodyBytes=800000000

**Comment by abdulsaheel** [comment]
If I'm not mistaken, you should be able to achieve this by navigating to:
Servers > localhost > Proxy > Configuration.

Then, add Your line to the configuration file and save.

**Reply by NubeBuster** [reply]
I added 

      - traefik.http.middlewares.my-requests.maxRequestBodyBytes=800000000
      
      To the labels section and restarted everything. I still get a 413 http response

**Reply by NubeBuster** [reply, marked as answer]
I found the issue. It was cloudflare dns proxy which has a file limit of 100MB.
